code/modules/jobs/job_types/roguetown/yeomen/archivist.dm

```diff
@@ -1,28 +1,33 @@
 /datum/job/roguetown/archivist
-	title = "Archivist"
-	tutorial = " The Archivist meticulously preserves and organizes ancient scrolls and tomes, safeguarding the collective knowledge of the realm for generations to come. Nobles and Peasants alike often seek the Archivists expertise on matters of history and fact."
+	title = "Scholar"
+	tutorial = "You've spent lots of time reading scrolls and tomes, gradually becoming a walking encyclopedia. \
+	Scholars meticulously preserve and organize ancient texts, safeguarding the collective knowledge of the realm for generations to come. \
+	Nobles and Peasants alike often seek the Scholars expertise on matters of history and fact."
 	flag = ARCHIVIST
 	department_flag = YEOMEN
 	faction = "Station"
 	total_positions = 1
 	spawn_positions = 1
-	spells = list(/obj/effect/proc_holder/spell/invoked/projectile/fetch)
-	allowed_races = RACES_TOLERATED_UP
+	allowed_races = RACES_SHUNNED_UP
```

Comment:
nah

Reply:
The scholar is a yeomen and not a courtier so allowing shunned races to play it is more than justified
![image](https://github.com/user-attachments/assets/72c026f5-66f3-47ae-8a35-e5abb9f23b49)
![image](https://github.com/user-attachments/assets/70143528-f97f-493f-b3bd-70ae522a2e02)
![image](https://github.com/user-attachments/assets/ec451ad3-a404-49df-8fc8-c8223657c91c)
![image](https://github.com/user-attachments/assets/38e18b93-268b-448d-bbeb-bd110029a9bd)
![image](https://github.com/user-attachments/assets/362412c9-040a-4f8d-9185-69f06aa96c21)

Reply:
if he lives in the keep he shall remain tolerated up

Reply:
As i remember the architect lives in the keep too.
![image](https://github.com/user-attachments/assets/9743c6be-35f5-4a03-a86e-8f3aff00fd59)

Reply:
I actually might add them a place outside the keep